### ISSUE TITLE
turf.intersect() endless loop！

### ISSUE THREAD

**Issue by xiaobebe** [Turfjs/turf#1481]
```
 const vecs1 = [
          [14.673293555299963, -447.5487838881006],
          [-56.03738456335748, -518.2594620067526],
          [-88.21074310734416, -486.0861034627635],
          [-17.50006498868672, -415.3754253441115],
        ];

 const vecs2 = [
          [178.42278626882, -415.3754253441115],
          [184.42278626882, -409.37541920011],
          [184.42278626882, -403.3754130561085],
          [-66.943427925018, -403.3754130561085],
          [-66.943427925018, -409.37541920011],
          [-66.943427925018, -415.3754253441115],
        ];


```

when i use this polygon vectors to turf.intersect(). endless loop 

![image](https://user-images.githubusercontent.com/19386356/45205318-626d7c80-b2b4-11e8-8f9b-377c3fb17235.png)

**Comment by rowanwins** [comment]
Hi @xiaobebe 

The easy solution for the timebeing is probably to roll back to one of the v5 series of `@turf/intersect`, and this should be resolved in the alpha release of `v7`

Cheers

**Comment by xiaobebe** [comment]
I found this problem is in `martinez-polygon-clipping` package. it has been resolved in `martinez-polygon-clipping@0.5.0 `.

the @turf/intersect dependencies is `"martinez-polygon-clipping": "^0.4.3"`.  so can you update for this.

**Comment by rowanwins** [comment]
Thanks for reporting back @xiaobebe

**Comment by xiaobebe** [comment]
can you update `martinez-polygon-clipping`?

my project use this

**Comment by rowanwins** [comment]
Resolved in 6.2.0 onwards. And by resolved in returns null.
